diff --git a/setup.py b/setup.py
index 62d218c..d0e3e44 100644
--- a/setup.py
+++ b/setup.py
@@ -6,12 +6,10 @@ except ImportError:
     from distutils.core import setup, find_packages

 setup(name='parcels',
-      version='1.0',
+      version='1.0.3',
       description="""Framework for Lagrangian tracking of virtual
       ocean particles in the petascale age.""",
       author="oceanparcels.org team",
-      use_scm_version=True,
-      setup_requires=['setuptools_scm'],
       packages=find_packages(),
       package_data={'parcels': ['include/*',
                                 'examples/*']},
       )
