--- setup.py
+++ setup.py
@@ -1,21 +1,18 @@
 """Install Parcels and dependencies."""

 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
-    from distutils.core import setup
+    from distutils.core import setup, find_packages

 setup(name='parcels',
-      version='0.0.1',
+      version='1.0',
       description="""Framework for Lagrangian tracking of virtual
       ocean particles in the petascale age.""",
       author="Imperial College London",
-      use_scm_version=True,
-      setup_requires=['setuptools_scm'],
-      packages=['parcels'],
-      package_data={'parcels': ['parcels/include/*',
-                                'parcels/examples/*']},
-      include_package_data=True,
+      packages=find_packages(),
+      package_data={'parcels': ['include/*',
+                                'examples/*']},
       entry_points={'console_scripts': [
           'parcels_get_examples = parcels.scripts.get_examples:main']}
       )
